.github/workflows/pr-test.yml

```diff
@@ -39,32 +43,20 @@ jobs:
         run: |
           yarn --frozen-lockfile
 
-      - name: Config git core.quotePath
+      - name: Get Changed files
         run: |
-          # If you don't do this, the get-diff-action won't be able to
-          # notice files that contain non-ascii characters.
-          # I.e.
-          #
-          #    ▶ git status
-          #    Changes not staged for commit:
-          #    ...
-          #   modified:   "files/en-us/glossary/b\303\251zier_curve/index.html"
-          #
-          # But after you set `core.quotePath` you get:
-          #
-          #    ▶ git status
-          #    Changes not staged for commit:
-          #    ...
-          #   modified:   "files/en-us/glossary/bézier_curve/index.html"
-          #
-          # Now, this gets used by the `git diff ...` inside get-diff-action.
-          git config --global core.quotePath false
-
-      - uses: technote-space/get-diff-action@v6
-        id: git_diff_content
-        with:
-          PATTERNS: files/**/*.+(html|md)
-          SET_ENV_NAME: GIT_DIFF_CONTENT
+          # Use the GitHub API to get the list of changed files
+          # documenation: https://docs.github.com/rest/commits/commits#compare-two-commits
+          gh api repos/{owner}/{repo}/compare/${{ env.BASE_SHA }}...${{ env.HEAD_SHA }} \
+            --paginate \
+            --jq '.files | .[] | select(.status|IN("added", "modified", "renamed", "copied", "changed")) | .filename' > ${{ env.CHANGED_FILES }}
```

Comment:
For there maybe too many files has been changed in a single PR, the `gh pr view` command used before can only fetch up to 100 changed files. But for almost all the PR, the commit in it would be less than `30` commits. So we can use github rest API to get changed files. (See <https://docs.github.com/rest/commits/commits?apiVersion=2022-11-28#compare-two-commits>). This API will limit 30 commits per page. So for those must few PRs, we can use `--paginate` (see options in <https://cli.github.com/manual/gh_api>) to fetch all the commits.

Addition: The [API limitation](https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#requests-from-github-actions) from GitHub Actions with `GitHub Token` would be 1000 requests per hour. This should be enough for most of cases.

Reply:
We can check this PR as an example: https://github.com/mdn/translated-content/pull/9513

PR info: https://api.github.com/repos/mdn/translated-content/pulls/9513
Compare info (all the 183 files in in the single rest response, see field `files`): https://api.github.com/repos/mdn/translated-content/compare/a9df62bb4885e5d9cdccd3834d5a24c921ef7cfa...16f5fc0db4ab1f5cb3858ab7878c428c19e9a79e

Reply:
If this is ok. Should we try it in [Markdown lint (PR)](https://github.com/mdn/translated-content/actions/workflows/pr-check_markdownlint.yml) first? There are some failed workflows (cannot get changed files), and it is less important than the current CI.